### PR TITLE
changed the text for reset password

### LINKED ISSUE #15529: spelling mistake in english reset password screen
image

Spellingerror in reset password screen, it should say "sent"  instead of  "send"

### PR DESCRIPTION
**Pull request by akshala**
Fix #15529

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

File: core/src/components/login/ResetPassword.vue [hunk 1]
```diff
@@ -49,7 +49,7 @@
 		</div>
 		<p v-if="message === 'send-success'"
 		   class="update">
-			{{ t('core', 'We have sent a password reset e-mail to the e-mail address known to us for this account. If you do not receive it within a reasonable amount of time, check your spam/junk folders.') }}
+			{{ t('core', 'A password reset message has been sent to the e-mail address for this account. If you do not receive it, check your spam/junk folders then ask your local administrator for help.') }}
```
Comment (juliushaertl) [inline core/src/components/login/ResetPassword.vue]:
```suggestion
			{{ t('core', 'A password reset message has been sent to the e-mail address of this account. If you do not receive it, check your spam/junk folders or ask your local administrator for help.') }}
```

Comment (rullzer) [reply core/src/components/login/ResetPassword.vue]: @akshala could you update your PR?

Comment (akshala) [reply core/src/components/login/ResetPassword.vue]: > @akshala could you update your PR?

yes okay

Comment (akshala) [reply core/src/components/login/ResetPassword.vue]: yes okay.

**Comment by skjnldsv** [comment]
```
modified: core/js/dist/login.js
291 | modified: core/js/dist/login.js.map
292
```

/compile amend /core/js/dist

**Pull request merged**